OpenRA.Mods.Common/Traits/Render/WithSpriteBody.cs

```diff
@@ -69,6 +72,15 @@ public string NormalizeSequence(Actor self, string sequence)
 			return RenderSprites.NormalizeSequence(DefaultAnimation, self.GetDamageState(), sequence);
 		}
 
+		public virtual void BuildingComplete(Actor self)
```

Comment:
Can you change this to `AddedToWorld()`, using the `INotifyAddedToWorld` interface instead of `INotifyBuildComplete`?